core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -263,13 +263,13 @@
       "The maximum size for each write-ahead log. See comment for property tserver.memory.maps.max"),
   TSERV_WALOG_MAX_AGE("tserver.walog.max.age", "24h", PropertyType.TIMEDURATION, "The maximum age for each write-ahead log."),
   TSERV_WALOG_TOLERATED_CREATION_FAILURES("tserver.walog.tolerated.creation.failures", "50", PropertyType.COUNT,
-      "The maximum number of failures tolerated when creating a new WAL file within the period specified by tserver.walog.failures.period."
```

Comment:
These removed docs mention a `tserver.walog.failures.period` prop.  I thought I might see something in this PR removing that prop, but did not.

Reply:
Precisely, that property does not exist.